python/chronos/src/bigdl/chronos/data/tsdataset.py

```diff
@@ -836,17 +837,21 @@ def to_tf_dataset(self, batch_size=32, shuffle=False):
 
     def to_numpy(self):
         '''
-        Export rolling result in form of a tuple of numpy ndarray (x, y).
+        Export rolling result in form of a numpy ndarray or a tuple of numpy ndarray (x, y)
+        or (x, y, x_enc, y_enc).
 
-        :return: a 2-dim tuple. each item is a 3d numpy ndarray. The ndarray
-                 is casted to float32.
+        :return: a 3d numpy ndarray or 2-dim or 4-dim tuple.
+                 each item of tuple is a 3d numpy ndarray.
+                 The ndarray is casted to float32.
         '''
         from bigdl.nano.utils.log4Error import invalidInputError
         if self.numpy_x is None:
             invalidInputError(False,
                               "Please call 'roll' method "
                               "before transform a TSDataset to numpy ndarray!")
-        if self.numpy_x_timeenc is None:
+        if self.numpy_y is None:
```

Comment:
what if label_len=0 (do we need to consider this circumstances?)

Reply:
Good catch. will change  `if self.numpy_y is None:` to `if self.numpy_y is None and self.numpy_x_timeenc is None:`